### DISCUSSION TITLE
Discord invite link, The oh no's and preyblade blast compound.

### DISCUSSION THREAD

**Discussion by Hesto01** [bhizusryaaajyaqiu/MindustryExpansion#45] [Q&A]
Just like it says in the title, I have some questions. The Discord invite link doesn't work. I thought I saw that te oh no's were being worked on, But I'm not sure. And the prey blade requires blast compound, which for some reason can't be excepted in the core, but a while back we could get the preyblade? I don't know how or what. I hope you see this and you fix the discord invite link so I can ask it all there

**Comment by Hesto01** [comment]
And I had some questions about the old version weapons like the railgun and Sni-shadow I think it was called? They were gone after a few updates so I was wondering

**Reply by sryaarf** [reply]
Because we prepare new update, pleasw you can wait it if we annouced at discord

**Comment by sryaarf** [comment]
This the new links for discord invites:

[Discord](https://discord.com/invite/CvrXTrhCjh)
